cf_units/__init__.py

```diff
@@ -191,8 +191,8 @@ def suppress_errors():
         except _ud.UdunitsError as e:
             error_msg = ': "%s"' % e.error_msg() if e.errnum else ""
             raise OSError(
-                "[%s] Failed to open UDUNITS-2 XML unit database%s"
-                % (e.status_msg(), error_msg)
+                f"[{e.status_msg()}] "
+                f"Failed to open UDUNITS-2 XML unit database{error_msg}"
```

Comment:
Looks to me that there should be a space after database (it wasn't there before though either)

Reply:
Actually `error_msg` starts with a colon, so I think it's right as-is.